### PR TITLE
Fix: Issue-2035 - Corrected Changlogs for multiple components

### PR DESCRIPTION
**Pull request by bhushan-ebi**

### CODE REVIEW

File: components/vf-collapse/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-### 0.1.1
+### 0.0.1
 
 * Deprecated `vf-collapse` component. `vf-details` component needs to be used instead. [Tracking issue](https://github.com/visual-framework/vf-core/issues/1911)
+* Changelog correction [Tracking issue](https://github.com/visual-framework/vf-core/issues/2035)
 
-### 0.1.0
+### 0.0.2
```
Comment (sandykadam) [inline components/vf-collapse/CHANGELOG.md:6]: Version order is different then other components

File: components/vf-explainer/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
-### 0.1.1
+### 0.0.1
 
 * Deprecated `vf-explainer` component. `vf-details` component needs to be used instead. [Tracking issue](https://github.com/visual-framework/vf-core/issues/1920)
 
-### 0.1.0
+### 0.0.2
```
Comment (sandykadam) [inline components/vf-explainer/CHANGELOG.md:6]: Same here version order is different

File: components/vf-collapse/CHANGELOG.md (hunk 1 above)
Comment (bhushan-ebi) [reply components/vf-collapse/CHANGELOG.md:6]: Fixed the order.

File: components/vf-explainer/CHANGELOG.md (hunk 2 above)
Comment (bhushan-ebi) [reply components/vf-explainer/CHANGELOG.md:6]: Fixed the order.

**Review by sandykadam — APPROVED** [review]

**Pull request merged**